docs(file-input): add file input topic (vnext) (#1487)
IgniteUI/igniteui-xplat-docs: doc/en/components/inputs/file-input.md

```

For a complete introduction to the {ProductName}, read the [*Getting Started*](../general-getting-started.md) topic.

## Configuration

### Properties

The File Input component offers a variety of properties that allow you to configure its behavior based on specific requirements. These properties give you control over the input’s functionality, appearance, and validation.

* `value` - Sets the current value of the file input field.
* `disabled` -	Disables the file input, preventing user interaction.
* `required` -	Marks the input as mandatory. Form submission will be blocked unless a file is selected.
* `invalid` -	Indicates that the input value is invalid, used to trigger visual error states.
* `multiple` -	Allows the selection of multiple files.
* `accept` -	Defines the types of files that can be selected. The value for this property needs to be a comma-separated list of file formats (e.g., .jpg, .png, .gif).
* `autofocus` -	Automatically focuses the file input field when the page loads.
* `label` -	Sets the label text associated with the file input element.
* `placeholder` -	Provides placeholder text displayed when no file is selected.

> 

Reviewer: The formatting of this is all over the place. Please fix it. Make sure that the spaces between the dashes and the text is the same for all items.

* `file-selector-text` - Allow you to customizes the text displayed on the file selection button.
* `file-missing-text` - Sets the text shown in the input field when no file has been selected.
* `value-missing` -  Renders custom content when the required field validation fails. (i.e., when a file is required but not provided).
* `invalid` – Allows you to render custom content when the input is in an invalid state.
* `custom-error` - Displays content when a custom validation message is set using the `setCustomValidity()` method.

`sample="/inputs/file-input/preffix-suffix", height="120", alt="{Platform} File Input Slots Example"`

## Integration

The File Input component integrates seamlessly with the HTML Form element. Using the methods and properties described above, you can effectively manage its behavior and validation within the standard HTML Forms.

`sample="/inputs/file-input/form", height="120", alt="{Platform} File Input Form Example"`

## Limitations

The File Input component currently has the following limitations:

> - The default text for the "Browse" button and the "No file chosen" message is not automatically localized. These texts remain the same across all locales but can be manually customized using the appropriate slots or placeholder binding.

Reviewer: ```suggestion
- The default strings for the "Browse" button and the "No file chosen" message are not automatically localized. These strings remain the same across all locales but can be manually customized using the appropriate slots or placeholder binding.
```

- Files cannot be set manually through the `value` property. File selection can be done only via the file picker.

> - When the `multiple` property is set, only the first selected file will be shown in the element value.

Reviewer: ```suggestion
```

This is not true. We attempt to show all item names. However, if the first item name is too long, the text will be truncated.
